### PR TITLE
@nx/react:library - Add simpleName option to remove dir name from generated file names

### PR DESCRIPTION
**Pull request by sir-captainmorgan21**
## Current Behavior
Right now if you provide a directory to place your library, it will result in the file name of the library components to include the directory path as a prefix. (IE: MyDirMyLibName.tsx).

## Expected Behavior
Expected behavior would be to allow a way to generate a simple file name only consisting of the name of the lib provided. Just like the `simpleName` option that exists for @nx/angular and was added to @nx/nest

## Related Issue(s)
https://github.com/nrwl/nx/issues/17207

### CODE REVIEW

**Comment by sir-captainmorgan21** [comment]
@mandarini tagging ya as requested

**Review by mandarini — CHANGES_REQUESTED** [review]
Just one small comment! Otherwise looks good to me! Thank you @sir-captainmorgan21 !

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,5 @@ CHANGELOG.md
 .env
 
 *.node
+
+.pnpm-store
```
[the remark this answers is not in the document]
Comment (mandarini) [reply .gitignore:37]: Hmmm I see... I was not getting that on my end, weird...

**Review by jaysoo — APPROVED** [review]

**Review by mandarini — APPROVED** [review]

**Review by mandarini — APPROVED** [review]

**Review by isaacplmann — APPROVED** [review]

**Review by jaysoo — APPROVED** [review]

**Comment by sir-captainmorgan21** [comment]
@mandarini thanks! how does the actual release process work in terms of seeing this get published? Do you queue up a small set of features/bug fixes?

**Comment by christopher-francisco** [comment]
Pure coincidence, I was just looking to see if this existed today, and it was merged yesterday. Where can I get info on the release process?

**Comment by mandarini** [comment]
Hi there @sir-captainmorgan21 @christopher-francisco ! We usually do patch release whenever there's a good amount of bug-fixes, but that depends. We just released `16.3.0`, I think maybe by the end of the week or sometime early next week? It all depends! :)

**Pull request merged**